cmd/krel/cmd/release_notes.go

```diff
@@ -85,6 +96,38 @@ func init() {
 		"version tag for the notes",
 	)
 
+	releaseNotesCmd.PersistentFlags().StringVarP(
```

Comment:
nit: Use `StringVar` if we have no shorthand

Reply:
DOne, all instances without short options are now StringVar